airbyte-api/src/main/openapi/config.yaml

```diff
@@ -165,6 +165,29 @@ paths:
           description: Workspace not found
         "422":
           $ref: "#/components/responses/InvalidInput"
+  /v1/workspaces/trynotification:
```

Comment:
This function should take a notification object and the test should be independent of the workspace and the fact the object already exist.

Reply:
Ok got it!